src/prefect/tasks/shell.py

```diff
@@ -110,6 +115,10 @@ def run(self, command: str = None, env: dict = None) -> str:
                     sub_process.returncode, line
                 )
                 self.logger.error(msg)
+
+                if self.log_stderr:
+                    self.logger.error(lines)
```

Comment:
Perhaps join these with `"\n"`? Lines is a list, and will be repr'd in the logs, which looks a bit odd:

```
logger.info(["this", "looks", "weird"])
['this', 'looks', 'weird']
```


I might also merge this with the `Command failed...` log above, and drop the `line` from the format message (it appears `line` is the last line of stdout, which feels a bit odd to selectively choose :shrug:)